cmd/libs/go2idl/make-client/output/doc.go

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
This is silly to put in a separate file. Can we just put it in the one output file?